src/features/stats/index.js

```diff
@@ -38,9 +39,33 @@ class Stats extends Component {
 
     render() {
         const { disabled, stats, sideMenu, largeView } = this.props;
-        const { abilities, level, exp, expToLevel, hp, maxHp, gold } = stats;
+        const {
+            abilities,
+            level,
+            exp,
+            expToLevel,
+            hp,
+            maxHp,
+            mana,
+            maxMana,
+            gold,
+            start,
+        } = stats;
         const { statsBgColor } = this.state;
 
+        let STRbonus = calculateModifier(abilities.strength);
+        let CONbonus = calculateModifier(abilities.constitution);
+        let DEXbonus = calculateModifier(abilities.dexterity);
+        let CHRbonus = calculateModifier(abilities.charisma);
+        let INTbonus = calculateModifier(abilities.intelligence);
+        let WISbonus = calculateModifier(abilities.wisdom);
```

Comment:
These variable names should follow camel case, something like `strBonus`

Reply:
Change done